src/Frontend/Modules/FormBuilder/Widgets/Form.php

```diff
@@ -521,7 +525,11 @@ private function validateForm()
                 // loop all fields
                 foreach ($this->item['fields'] as $field) {
                     // skip
-                    if ($field['type'] == 'submit' || $field['type'] == 'paragraph' || $field['type'] == 'heading') {
+                    if ($field['type'] == 'submit' ||
```

Comment:
I don't agree
if you are saying a list outloud you also dont say I have this and, this and, this
you say I have this, and this, and this